libs/base/ui/form/input-list/src/input-list.styles.ts

```diff
@@ -15,7 +15,7 @@ export const styles = css`
 
     display: none;
     margin-block-end: 8px;
-    color: var(--oryx-color-ink);
+    color: var(--oryx-color-neutral-12);
```

Comment:
looks like `inkColor`

Reply:
I have dropped the color all together, so that the ink color is inherited. I've also left a note on the Figma to get better inputs on the color (think it should be ink, but it's not clear)